gui_qt/toast_widget.py

```
@@ -37,31 +39,28 @@ class ToastStyle(Enum):
 _ICON_TEXT: dict[ToastStyle, str] = {
-    ToastStyle.SUCCESS: "✔",
-    ToastStyle.WARNING: "⚠",
-    ToastStyle.ERROR: "✘",
+    ToastStyle.SUCCESS: "✓",
+    ToastStyle.WARNING: "!",
+    ToastStyle.ERROR: "×",
 }
 
+# Strip caller-supplied status glyphs so we never show icon + message glyph twice.
+_LEADING_STATUS_GLYPHS = re.compile(
+    r"^[\s✓✔✕✖❌⚠⚠️✘×!！]+",
```

Ruff (0.15.20)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)